.github/workflows/development-environment.yml

```diff
@@ -13,16 +13,15 @@ jobs:
   dev-environment:
     name: python set up
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 20
+    timeout-minutes: 25
     strategy:
       matrix:
-        os: [ macos-latest ]
+        os: [ macos-latest, macos-11.0, ubuntu-latest ]
```

Comment:
Can we add a comment to clarify what the different os tags mean? at least for macos